src/clj/game/core/ice.clj

```diff
@@ -534,6 +534,16 @@
               (map :label (sort-by :index broken-subs)))
         "\")")))
 
+(defn break-subs-event-context
+  [state _ ice broken-subs breaker]
```

Comment:
Given that this isn't an async function, you can just drop the `side` arg. No need to carry that baggage around if you're not going to write `(wait-for (break-subs-event-context ...)`